package.json

```diff
@@ -36,5 +36,10 @@
   },
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "eslintConfig": {
```

Comment:
to be removed - this should be in another PR, when all ts issues will be resolved